x-pack/legacy/plugins/uptime/public/components/functional/location_map/location_missing.tsx

```diff
@@ -26,13 +32,20 @@ export const LocationMissingWarning = () => {
 
   const button = (
     <EuiButton iconType="alert" size="s" color="warning" onClick={togglePopover}>
-      Geo Information Missing
+      <FormattedMessage
+        id="xpack.uptime.locationMap.locations.missing.title"
+        defaultMessage="Geo Information Missing"
+      />
     </EuiButton>
   );
 
   return (
-    <EuiFlexGroup data-test-subj="xpack.uptime.locationMap.locationMissing" gutterSize="none">
-      <EuiFlexItem grow={false} style={{ marginLeft: 'auto', marginRight: 20 }}>
+    <EuiFlexGroup
+      data-test-subj="xpack.uptime.locationMap.locationMissing"
+      gutterSize="none"
+      responsive={false}
+    >
```

Comment:
I would do it that way (no idea if this is a good way), so the button is always on the right
```
<div data-test-subj="xpack.uptime.locationMap.locationMissing"
style={{display: 'flex', flexDirection: 'row-reverse'}}
```
Let me know what you think

Reply:
i think both are perfectly fine :)